osu.Game/Overlays/Changelog/ChangelogHeader.cs

```diff
@@ -21,16 +24,16 @@ public class ChangelogHeader : BreadcrumbControlOverlayHeader
 
         public ChangelogUpdateStreamControl Streams;
 
-        private const string listing_string = "listing";
+        public static LocalisableString ListingString => LayoutStrings.HeaderChangelogIndex;
```

Comment:
@smoogipoo thoughts on this one? is used in tests (`public` part of the weirdness) but not sure if we want this, or another case of just inlining the references at each point.

Reply:
I think I'm fine with this. I just don't like redirects in the supposed-to-be-autogenerated XStrings classes because the format isn't guaranteed to remain when new strings are added and I can't guarantee it won't break the analyser in the first place.

But this seems okay to me.